core/primitives/src/block.rs

```diff
@@ -123,9 +123,7 @@ impl Block {
         clock: near_time::Clock,
         sandbox_delta_time: Option<near_time::Duration>,
         optimistic_block: Option<OptimisticBlock>,
-        // TODO(spice): Remove option once spice feature is tied to protocol version. We use option
-        // for now to indicate whether spice feature is enabled to allow creating non-spice blocks
-        // for tests.
+        // Spice block is created IFF this is Some.
```

Comment:
I guess we still want to remove the Option at the end of the project after everything is SPICE only?